build_projects/dotnet-cli-build/GenerateGuidFromName.cs

```diff
@@ -0,0 +1,82 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.IO.Compression;
+using System.Runtime.InteropServices;
+using System.Net.Http;
+using Microsoft.Build.Utilities;
+using Microsoft.DotNet.Cli.Build.Framework;
+using Microsoft.DotNet.InternalAbstractions;
+using Microsoft.Build.Framework;
+using Microsoft.Build.Utilities;
+using System.Security.Cryptography;
+
+using static Microsoft.DotNet.Cli.Build.Framework.BuildHelpers;
+
+namespace Microsoft.DotNet.Cli.Build
+{
+    public class GenerateGuidFromName : Task
```

Comment:
Why can't we delete Utils.GenerateGuidFromName, and move over the 2 callers of it to use GenerateGuidFromName.GenerateGuid()?

---
In reply to: [68872777](https://github.com/dotnet/cli/pull/3696#discussion_r68872777) [](ancestors = 68872777)